src/main/java/com/robertx22/age_of_exile/event_hooks/my_events/OnLootChestEvent.java

```diff
@@ -23,16 +31,28 @@ public void accept(ExileEvents.OnChestLooted event) {
 
         LootInfo info = LootInfo.ofChestLoot(player, event.pos);
 
+        Load.player(player).favor.onLootChest(player);
```

Comment:
My opinion here is:

Just because the chest generates no loot shouldn't mean it doesn't grant you favor. So I bumped this up a bit.